### PR TITLE
Use ansible_facts instead of global scoped variables

### PR DESCRIPTION
**Pull request by evrardjp**

### CODE REVIEW

**Review by noonedeadpunk — COMMENTED** [review]
LGTM

File: tasks/main.yml [hunk 1]
```diff
@@ -29,8 +34,8 @@
     file: keepalived_selinux.yml
   when:
     - keepalived_selinux_compile_rules | length > 0
-    - ansible_selinux.status is defined
-    - ansible_selinux.status == "enabled"
+    - ansible_facts['selinux']['status'] is defined
```
Comment (noonedeadpunk) [inline tasks/main.yml:37]: Oh, well, I'd probably did smth like

- "'selinux' in ansible_facts"

Comment (evrardjp) [reply tasks/main.yml:37]: it is always in ansible_facts, even without selinux. Status should be defined, but I kept it for safety.

Comment (noonedeadpunk) [reply tasks/main.yml:37]: Huh, I thought that for Debian it should not be there...

**Pull request merged**